ts/src/binance.ts

```diff
@@ -5245,35 +5396,38 @@ export default class binance extends Exchange {
         const initialUppercaseType = type.toUpperCase ();
         const isMarketOrder = initialUppercaseType === 'MARKET';
         const isLimitOrder = initialUppercaseType === 'LIMIT';
-        const postOnly = this.isPostOnly (isMarketOrder, initialUppercaseType === 'LIMIT_MAKER', params);
```

Comment:
I think the postOnly logic is broken 🤔 
![image](https://github.com/ccxt/ccxt/assets/43336371/4b0efc26-9edf-4bbc-9406-200a6fff79bc)